docs/src/main/asciidoc/opentracing.adoc

```diff
@@ -99,6 +99,8 @@ public class TracedResource {
 }
 ----
 
+<1> The `Logger` class is used in order to append tracing information to the console log output. Class which contains span and wanted to be logged in the console should define its `Logger` as well.
```

Comment:
This needs to be closely tied with the `traceId` log format in `application.properties`. Without the log format present, adding a logger to the code is not sufficient to make it work.

Reply:
Updated